docs/concepts/trading-on-vega/positions-margin.md

```diff
@@ -0,0 +1,133 @@
+
+There are a few mechanisms that work differently to how they would on a centralised exchange, in order to keep the markets solvent. They include:
+- [**Mark to market**](#mark-to-market): Mark to market on Vega happens much more frequently than on a centralised exchange. Every time the market price moves, the mark to market price is recalculated
+- [**Margin**](#margin): Vega has implemented automated [cross-margining](#cross-margining). Margin is calculated automatically depending on the number of positions, the size, and the market movements so that there's enough collateral available to sustain a position and not put other market participants, or the market itself, under strain
+
+## Mark to market
+Marking to market refers to settling gains and losses due to changes in the market value of the underlying product. Marking to market aims to provide a realistic appraisal of a position based on the current market conditions.
+
+When marking to market, the protocol takes the current market price and recalculates traders' margin requirements based on how their position is affected by price moves.
+
+If the market price goes up, a trader that holds a long position receives money in their margin account – equal to the underlying's change in value – from a trader that holds a short position, and conversely if the value goes down, the holder of the short position recieves money from the holder of the long position.
+
+For a futures market created on Vega, the mark-to-market price is calculated every time the price moves, and is based on the last traded price. This is in contrast to traditional futures markets, for which marking to market occurs once per day. One exception is when the market settles at expiry, at which point the mark to market price comes from the data source's final settlement price.
+
+Settlement instructions are generated based on the change in market value of the open positions of a party. When the mark price changes, the network calculates settlement cash flows for each party, and the process is repeated each time the mark price changes until the maturity date for the market is reached.
+
+Because the margin for a market is calculated dynamically based on the market conditions, the mark price also has an effect on how much collateral is set aside for margin.
+
+:::note Read more
+[Mark to market settlement](./settlement#mark-to-market-settlement)
+:::
+
+## Margin
+Margin is the amount of collateral required to keep your position open. It can change depending on how your position is impacted by your own actions and market movement. The margin calculations ensure a trader does not enter a trade that will immediately need to be closed out.
+
+The calculation for a new position, and the amount deducted from collateral to cover margin, is based on all of a trader's open orders. A trader will need enough margin to keep a position open, whether it goes for or against the trader.
```

Comment:
```suggestion
When placing a key's first order on a market: the protocol will calculate the initial margin required. If this there is not a sufficient balance in the general account to fund this, the order will be rejected. If there is these funds will be moved into a margin account for that market.  
```
Removed the stuff about other orders, because when opening a position there wouldn't be any.